testsuite/features/step_definitions/command_steps.rb

```diff
@@ -190,7 +190,11 @@
 
 When(/^vendor change should be enabled for "(?:[^"]*)" on "([^"]*)"$/) do |host|
```

Comment:
This method has two regular expressions, but only one parameter.
I guess we should have filter by the first text or something, not sure, but it seems wrong as it is.

Reply:
The first regexp is not captured: it is inside a shy group.

Reply:
I don't get why we need it then

Reply:
The reason is that we use this step at 3 feature files: the matched string inside the shy group is different (it serves as documentation, in the sense that reflects the action we are doing in the context where the step is used). On the other hand, we use a shy group because we don't need such info inside the body of the step.
It's just a matter of style; we could instead capture the string and define a local var `_foo` in the block: `|_foo, host|`.
I prefer the shy group than pollute with an unused variable.

Reply:
In that case, could we please don't use the double quotes?

Reply:
Agreed.

Reply:
We agreed on fixing it out of this PR, as this is out of scope.

Reply:
Done here https://github.com/uyuni-project/uyuni/pull/3605